src/types/mod.rs

```diff
@@ -76,3 +109,73 @@ pub enum CreateMode {
     //  ^---- is it sequential?
     //   ^--- is it ephemeral?
 }
+
+#[derive(Clone, Copy, Debug, PartialEq)]
+pub enum ZkConnectStringError {
```

Comment:
Add a comment for this type similar to the one above for `CreateMode` so that a description shows up in the generated docs.